### ISSUE TITLE
Inconsistent pop-up behavior on different laptop/browser - Windows 10 Scaling

### ISSUE THREAD

**Issue by rneppell** [afackler/FireEyes#11]
I'm getting some oddly inconsistent behavior between two different laptops. I was using my HP Spectre Win10 laptop with Chrome as my main work machine, but a few days ago switched to a Surface Book 2 Win10 tablet.  

On the SB2, pop-ups are appearing basically as I'd expect (with the exception of some odd font size issues). 

But on the Spectre, I'm seeing them behave completely differently: they seem to only display a generic "IRWIN FIRE LOCATION" pop-up, and then if the user clicks the right arrow displays a full-screen pop-up with the actual fire info. 

Attaching screenshots from both machines here. The HP Spectre has screen resolution of 2560 x 1440 and I had Windows 10's "Scaling" set to 200% (which Win10 tells me is recommended for my screen). 

But AHA! I just changed the scaling to 100% and magically the map pop-ups now start behaving like expected / same as my Surface Book 2.  When I change the scaling back to 200% --- they go back to the weirdness I show in the hp_spectre screenshots. 

Furthering the fun, I also have the SB2's scaling set to 200%, so it's not *just* that value that's causing different behavior. The SB2 is hooked up to my good monitor now, and that has 4k (3840x2160) resolution so I'm guessing that's also a factor. 

I'm attaching the hp_spectre screenshots (at 200% scaling) here and hitting submit; will then switch machines and take screenshots on the SB2 and attach in comment...
![image](https://user-images.githubusercontent.com/44247905/84402686-9f695000-abb9-11ea-87b9-30ee7e5bef91.png)

![image](https://user-images.githubusercontent.com/44247905/84402823-d2134880-abb9-11ea-918c-0862ee200950.png)

**Comment by rneppell** [comment]
Here's how the pop-up displays on my Surface Book 2 running Win10 (screen resolution 3840x2160, Windows scaling 200%). 

(To me at least this is how it *should* look / is correct)

![image](https://user-images.githubusercontent.com/44247905/84403570-b3618180-abba-11ea-8dc2-a7eddea59391.png)

**Comment by rneppell** [comment]
However, even on the SB2 sometimes the pop-ups display with a strangely large font. 

I haven't been able to figure out a pattern of which points give me a "normal" font size pop-up vs. the big size, but right now it seems like about 50-50 (half seem to display as expected, half not...)

![image](https://user-images.githubusercontent.com/44247905/84403925-194e0900-abbb-11ea-84f1-e22aef269bed.png)

**Comment by rneppell** [comment]
...it may be that it is the pop-ups that have only a small amount of text in them that are getting scaled up to big font, but not 100% sure of that.  If so that may be "works as designed" / not a big deal...

**Comment by rneppell** [comment]
A bit more info; definitely the Windows Scaling having an impact. On the SB2 I was at 200% scaling; I incrementally kept increasing it to see if/when it switched to the weird behavior I saw on the Spectre. 

It worked fine until I got to 300%, at which point sure enough it displayed the pop-ups in same style I saw on the Spectre. 

Incidentally: all tests on both machines were done on Chrome and the scaling within Chrome itself was left at the default 100% for both. 

![image](https://user-images.githubusercontent.com/44247905/84405044-75fdf380-abbc-11ea-953c-9e3e91a09ff2.png)
